create_rls.py

```diff
@@ -144,27 +137,36 @@ def upload_to_s3(file, s3_file):
 
 
 def main(separator=":"):
-    ou_tag_data = {}
-    root_ou = ROOT_OU
-    ou_tag_data = process_ou(root_ou, ou_tag_data, root_ou)
-    ou_tag_data = process_root_ou(root_ou,ou_tag_data)
-    print(f"OU_TAG_DATA: {ou_tag_data}")
-    qs_users = get_qs_users(ACCOUNT_ID, qs_client)
-    qs_users = {qs_user['UserName']: qs_user['Email'] for qs_user in qs_users}
-    qs_email_user_map = {}
-    for key, value in qs_users.items():
-        if value not in qs_email_user_map:
-            qs_email_user_map[value] = [key]
-        else:
-            qs_email_user_map[value].append(key)
-    qs_rls = {}
-    for entry in ou_tag_data:
-        if entry in qs_email_user_map:
-            for qs_user in qs_email_user_map[entry]:
-                qs_rls[qs_user] = ou_tag_data[entry]
-    print("QS EMAIL USER MAPPING: {}".format(qs_email_user_map))
-    print("QS RLS DATA: {}".format(qs_rls))
-    write_csv(qs_rls)
+ 
+    MANAGEMENT_ACCOUNT_IDS = os_environ['MANAGEMENT_ACCOUNT_IDS']
+
+    for payer_id in [r.strip() for r in MANAGEMENT_ACCOUNT_IDS.split(',')]:
+        org_client = assume_management(payer_id)
+        root_ou = org_client.list_roots()['Roots'][0]['Id']
+
+        qs_client = boto3.client('quicksight',region_name=QS_REGION)
+
+        ou_tag_data = {}
+        ou_tag_data = process_ou(org_client, root_ou, ou_tag_data, root_ou)
+        ou_tag_data = process_root_ou(org_client, root_ou,ou_tag_data)
+        print(f"OU_TAG_DATA: {ou_tag_data}")
+        qs_users = get_qs_users(ACCOUNT_ID, qs_client)
+        qs_users = {qs_user['UserName']: qs_user['Email'] for qs_user in qs_users}
+        qs_email_user_map = {}
+        for key, value in qs_users.items():
+            if value not in qs_email_user_map:
+                qs_email_user_map[value] = [key]
+            else:
+                qs_email_user_map[value].append(key)
+        qs_rls = {}
+        for entry in ou_tag_data:
```

Comment:
I had a bug here, replace this for cycle to 

        if entry in qs_email_user_map:
            for qs_user in qs_email_user_map[entry]:
                if qs_user not in qs_rls:
                    qs_rls[qs_user] = ou_tag_data[entry]
                else:
                    qs_rls[qs_user] = ou_tag_data[entry.extend(entry)]

Reply:
When i add this in its not working, can you pull down my update and add it and push to the branch please

Reply:
Done